doc/ref_cert/RC2/chapters/chapter01.md

```diff
@@ -1,5 +1,188 @@
+ - Verify, with requirements traceability, that the installation cookbooks (manifests) are in conformance with the RA-2 specifications (for example, software versions, plugins, and configurations).
+ - Guidelines for conformance testing evaluation criteria used for CNTT related infrastructure badging programs
+ - Guidelines for conformance testing environments and tools for enabling infrastructure vendor or 3rd party conformance testing
+
+In summary, **Conformance** testing will be performed as part of cloud infrastructure and CNF lifecycle testing which includes **Verification** and **Validation**, defined further as:
+
+- **Verification** confirms design requirement specifications are met using Requirement Traceability and Manifest Reviews to ensure the cloud infrastructure or CNF is delivered per implementation specifications.
+- **Validation** confirms the infrastructure or CNF meet the expected or desired behaviour.
+
+**All Terms utilized throughout this chapter are intended to align with LFN Complinace and Verification Committee (CVC) definitions, and their use through CVC documentation, guidelines, and standards.**
+
+<a name="1.1"></a>
+## 1.1 Introduction
+
+**Chapter Purpose**<br>
+This chapter includes process flow, logistics, and requirements which must be satisfied to ensure Kubernetes infrastructure meets the design, feature, and capability expectations of the Reference Model (RM) and RA2. Ultimately, RC2 will reduce the amount of time and cost it takes each operator to on-board and maintain cloud infrastructure and CNFs.
+
+**Objective**<br>
+Specify Kubernetes infrastructure and CNF Conformance aspects using requirements in CNTT RA2 and RM.
+
+Upstream projects define features/capabilities, test scenarios, and test cases which may be used for infrastructure and/or CNF verification and validation. Where feasible existing test frameworks, test methods and tests may also be used be used for conformance testing along with new tests developed where there are gaps.  
+
+Many existing tests currently being considered for RC-2 are functional tests from upstream projects. Reuse of existing test suites is encouraged however OVP 2.0 is not aiming to replace programs such as Certified Kubernetes which is a "CNCF software conformance program that ensures a vendor’s version of Kubernetes supports the required APIs, as do open source community version". 
+
+The focus of conformance is to test the actual behavior of the system with respect to a capability specified in the RM or RA-2. 
+
+
+**Test Methodology**
+- Verification test to make sure that the Kubernetes services have been deployed and configured correctly
+- Manifest Verifications (Termed Compliance by CVC) will ensure the infrastructure is compliant, and delivered for testing, with hardware and software profile specifications defined by the RM and RA2
```

Comment:
same as above: why only verification testing and not validation testing?

Reply:
added